scripts/createYoeoLabelsTORSO.py

```diff
@@ -127,134 +121,129 @@ def range_limited_float_type_0_to_1(arg):
-        with open(os.path.join(labels_dir, name + ".txt"), "w") as output:
-            output.writelines([annotation + "\n" for annotation in annotations])
+            elif annotation['type'] in CLASSES['segmentation_classes']:  # Handle segmentations
+                continue
+            elif annotation['type'] in CLASSES['ignored_classes']:  # Ignore this annotation
+                continue
+            else:
+                print(f"The annotation type '{annotation['type']}' is not supported or should be ignored. Image: '{img_name}'")
+
+    # Store BB annotations in .txt file
+    with open(os.path.join(labels_dir, name + ".txt"), "w") as output:
+        output.writelines([annotation + "\n" for annotation in annotations])
+
+# Create symlinks for images to destination directory
+# This is helpful, if dataset directory is read-only
+if create_symlinks:
+    for image_name in image_names:
+        link_path = os.path.join(images_dir, image_name)
+        target_path = os.path.join(dataset_dir, "images", image_name)
+        os.symlink(target_path, link_path)
 
 # Seed is used for train/test split
 random.seed(args.seed)
 print(f"Using seed: {args.seed}")
 
 # Generate train/testsplit of images
-random.shuffle(sorted(images))  # Sort for consistant order then shuffle with seed
-train_images = images[0:round(len(images) * (1 - args.testsplit))]  # Split first range
-test_images = images[round(len(images) * (1 - args.testsplit)) + 1:-1]  # Split last range
+random.shuffle(sorted(image_names))  # Sort for consistent order then shuffle with seed
+train_images = image_names[0:round(len(image_names) * (1 - args.testsplit))]  # Split first range
+test_images = image_names[round(len(image_names) * (1 - args.testsplit)) + 1:-1]  # Split last range
 
 # Generate meta files
 train_images = set(train_images)  # Prevent images from showing up twice
-train_path = os.path.join(args.superset, "train.txt")
+train_path = os.path.join(destination_dir, "train.txt")
 with open(train_path, "w") as train_file:
-    train_file.writelines([image_path + "\n" for image_path in train_images])
+    train_file.writelines([str(os.path.join(destination_dir, image_name)) + "\n" for image_name in train_images])
 
 test_images = set(test_images)  # Prevent images from showing up twice
-test_path = os.path.join(args.superset, "test.txt")
+test_path = os.path.join(destination_dir, "test.txt")
 with open(test_path, "w") as test_file:
-    test_file.writelines([image_path + "\n" for image_path in test_images])
+    test_file.writelines([str(os.path.join(destination_dir, image_name)) + "\n" for image_name in test_images])
 
-names_path = os.path.join(args.superset, "yoeo.names")
+names_path = os.path.join(destination_dir, "yoeo.names")
 with open(names_path, "w") as names_file:
     names_file.writelines([class_name + "\n" for class_name in CLASSES['bb_classes']])
 
-data_path = os.path.join(args.superset, "yoeo.data")
+data_path = os.path.join(destination_dir, "yoeo.data")
```

Comment:
It possibly too much for this PR, but we should drop the train test split and make the script compatible with the full torso format generating a valid data config that includes both test and train set by default.